dns/nextdns-community/src/etc/nextdns.conf

```diff
@@ -0,0 +1,17 @@
+config 4e6e51
```

Comment:
You should remove this file .. the template needs to take care of it :)

Reply:
how do I run the full init/config on the Save event on the form so the .config file can be generated? I tried to find the connection (what triggers what) on ntopng plugin and am still lost...